web/package.json

```diff
@@ -11,18 +11,20 @@
     "@mui/lab": "^5.0.0-alpha.84",
     "@mui/material": "5.13.2",
     "@mui/styles": "5.13.2",
```

Comment:
Remove this line @mul/styles as I hit issue when remove the node_modules directory and npm install

https://github.com/mui/material-ui/issues/32982

And then a few more place need to fix once this dropped

Reply:
I think it works now. Could you help me check it again?